modules/tile-converter/src/i3s-server/controllers/index-controller.ts

```diff
@@ -12,9 +12,16 @@ const I3S_LAYER_PATH = process.env.I3sLayerPath || '';
  */
 export async function getFileNameByUrl(url: string): Promise<string | null> {
   const extensions = ['json', 'bin', 'jpg', 'jpeg', 'png', 'bin.dds', 'ktx2'];
-  const FULL_LAYER_PATH = path.join(process.cwd(), I3S_LAYER_PATH);
+  const absolutePath = /^[^.]+/.exec(I3S_LAYER_PATH);
```

Comment:
Does it check paths for Windows (`c:\....`)?

Reply:
It should work with Windows path. Regex checks './' relative path.

Reply:
Does it checkt that string starts from something. Use `I3S_LAYER_PATH.startWith('./')` instead

Reply:
Added 'startWith' function to check path